pkg/plugin/linuxutil/netstat_stats_linux.go

```diff
@@ -275,8 +282,8 @@ func validateRemoteAddr(addr string) bool {
 		return false
 	}
 
-	// ignore localhost addresses.
-	if strings.Contains(addr, "127.0.0") {
+	// ignore localhost addresses and the node's own IP
+	if strings.HasPrefix(addr, "127.0.0") || addr == nodeIP || addr == "0.0.0.0" {
```

Comment:
Why the different ways to compare IPs? Maybe create a `denyList` of IPs and a `validateIgnore` or something similar that uses same string comparison method.